jupyter_sphinx/execute.py

```diff
@@ -587,13 +595,13 @@ def cell_output_to_nodes(cell, data_priority, write_stderr, dir, thebe_config):
 def attach_outputs(output_nodes, node, thebe_config, cm_language):
     if thebe_config:
         source = node.children[0]
-
         thebe_source = ThebeSourceNode(hide_code=node.attributes['hide_code'],
                                        code_below=node.attributes['code_below'],
                                        language=cm_language)
         thebe_source.children = [source]
 
         node.children = [thebe_source]
+        node.attributes["classes"] = ["jupyter_container"] # add jupyter classes even if thebe_config <-- CHECK
```

Comment:
Did you check what happens when thebelab is enabled? (I presume this is what the `CHECK` in the comment means)

Reply:
Sort of. I'm not quite sure what you all are looking for in the formatting for thebelab. I will test it again.

Reply:
I'm not sure either, but I saw the "CHECK" in the comment and thought it was something to mention.

Reply:
I have adjusted in the last pull request. It will now add the jupyter_container CSS if there is a hide-code directive regardless of whether thebelab is enabled.